health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/pages/employee/L1Main.js

```
@@ -0,0 +1,698 @@
+            <div>
+              <MultiLink
+                className="multilink-block-wrapper"
+                label={t(`ES_DSS_SHARE`)}
+                icon={<ShareIcon className="mrsm" />}
+                showOptions={(e) => setShowShareOptions(e)}
+                onHeadClick={(e) => setShowShareOptions(e !== undefined ? e : !showShareOptions)}
+                displayOptions={showShareOptions}
+                options={shareOptions}
+              />
+            </div>
+            <div onClick={handlePrint}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)